lib/syntax_tools/src/erl_syntax_lib.erl

```diff
@@ -23,9 +23,6 @@
 %%
 %% This module contains utility functions for working with the
 %% abstract data type defined in the module {@link erl_syntax}.
-%%
-%% @type syntaxTree() = erl_syntax:syntaxTree(). An abstract syntax
```

Comment:
I'm ok with this kind of changes as it removes the "see module …" and it's not too many changes.
But I'm not fond of introducing new type aliases.

If there was an attribute like `-import_type(module, [type/arity])` of which the semantics guarantee that the module-prefixed type and its alias are exactly the same (and not some union), then I would agree with aliasing types to make them shorter.
Well, even then I don't think I would use it myself.